flood_forecast/multimodal_models/prediff.py

```diff
@@ -0,0 +1 @@
+# PreDiff is on hold as it very complicated and will take a while to implement.
```

Comment:
Yes this will serve as placeholder until we figure out how to add stable diffusion models into FF